### PR TITLE
Update debugging

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update debugging:
- [x] Change 'aka' to 'also known as'

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)
Umbraco CMS v10, 12 & 13

### CODE REVIEW

File: 10/umbraco-cms/reference/debugging.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ To read about SourceLink, you can take a look at the following websites:
 * Install the latest Umbraco CMS 9.0+ Nuget Packages from Nuget.org
 * Create an IComposer or similar code in your new site/SLN that you want to F11/Step Into. [Example Code Snippet to try with SourceLink](debugging.md#example-code-snippet-to-try-with-sourcelink)
 * Prompt will appear and the original source code file is fetched directly from GitHub. ![Visual Studio 2019 SourceLink dialog](../../../11/umbraco-cms/reference/images/VS19-sourcelink-prompt.png)
-* How far can you `F11` aka `Step Into` and go down the rabbit hole of the Umbraco CMS source code?
+* How far can you `F11` also knonw as `Step Into` and go down the rabbit hole of the Umbraco CMS source code?
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/debugging.md:36]:
```suggestion
* How far can you `F11`, also known as `Step Into`, and go down the rabbit hole of the Umbraco CMS source code?
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR as always 😊 🙌 

I have one suggestion, can you please take a look into it and make the changes to all versions if you agree with it? 💡

File: 10/umbraco-cms/reference/debugging.md (hunk 1 above)
Comment (erikjanwestendorp) [reply 10/umbraco-cms/reference/debugging.md:36]: oops 😅

**Comment by erikjanwestendorp** [comment]
@alina-tincas Oops... 😅, just updated the PR 😄

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
No worries 🙈 

All looks good now, so this PR will be merged in Thursday😊

**Pull request merged**